src/lib/lightnet.js

```diff
@@ -13,6 +15,12 @@ const lightnetConfigFile = path.resolve(
   `${Constants.lightnetWorkDir}/config.json`
 );
 const lightnetLogsDir = path.resolve(`${Constants.lightnetWorkDir}/logs`);
+const lightnetExplorerDir = path.resolve(
+  `${Constants.lightnetWorkDir}/explorer`
+);
+const lightnetExplorerConfigFile = path.resolve(
+  `${lightnetExplorerDir}/config.json`
```

Comment:
The `config.json` file is already being used to store the deployAlias and other configurations.

```
{
  "version": 1,
  "deployAliases": {
    "berkeley": {
      "url": "https://proxy.berkeley.minaexplorer.com/graphql",
      "keyPath": "keys/berkeley.json",
      "feepayerKeyPath": "This is specific to each developer and can be generated with the zkApp-cli. Update with your feepayer keypath to run this tutorial example",
      "feepayerAlias": "dev",
      "fee": ".1",
      "smartContract": "Add"
    }
  }
}
```
You could add the lightnet explorer version to this existing `config.json` file. Are the lightnet and lightnet explorer configs scoped to a `deployAlias` or are they used across all `deployAliases`?  If it is not scoped to a deployAlias we could add a lightnet object that containes the lighnet explorer and lightnet configs.

```
{
  "version": 1,
  "deployAliases": {
    "berkeley": {
      "url": "https://proxy.berkeley.minaexplorer.com/graphql",
      "keyPath": "keys/berkeley.json",
      "feepayerKeyPath": "This is specific to each developer and can be generated with the zkApp-cli. Update with your feepayer keypath to run this tutorial example",
      "feepayerAlias": "dev",
      "fee": ".1",
      "smartContract": "Add"
    }
  },
  "lightnet": {
     "lightnetExplorerVersion: "0.XXX",
     "other config"
  }
}
```

Reply:
Thanks @ymekuria!
I'm not sure if we need this right now because with such approach `zk lightnet` will be linked to specific project but currently we're not providing the way to run local networks for every projects independently and because of this the idea of `zk lightnet` is to be accessible anywhere (not just within the zkapp project).
Adjusting it to what you're suggesting will require additional refactoring and testing (like check if we are within the zkapp project and if so configure stuff for specific project only but if we're not then configure it globally like it is right now).
https://github.com/o1-labs/zkapp-cli/issues/531